src/_data/sidenav.yml

```
@@ -151,6 +151,8 @@
       permalink: /cookbook/design/themes
     - title: Material design
       permalink: /ui/design/material
+   - title: Material 3 migration
```

I think this missing space is what is causing the build failure. Also consider a task based title here since it's a migration guide:

```suggestion
    - title: Migrate to Material 3
```